docs/charts/line.md

```diff
@@ -43,7 +43,7 @@ All point* properties can be specified as an array. If these are set to an array
 | `pointHoverRadius` | `Number/Number[]` | The radius of the point when hovered.
 | `showLine` | `Boolean` | If false, the line is not drawn for this dataset.
 | `spanGaps` | `Boolean` | If true, lines will be drawn between points with no or null data. If false, points with `NaN` data will create a break in the line
-| `steppedLine` | `Boolean` | If true, the line is shown as a stepped line and 'lineTension' will be ignored.
+| `steppedLine` | `Boolean/String` | If true, the line is shown as a stepped line and 'lineTension' will be ignored. For String values see [Step Interpolation](#stepInterpolation)
```

Comment:
I would go for the same strategy as above to keep the description in the table concise, maybe something like:

```javascript
| `steppedLine` | `Boolean/String` | whether the line is shown as a stepped line. [more...](#stepped-line)
```